app/javascript/dashboard/i18n/locale/en/inboxMgmt.json

```diff
@@ -83,7 +83,7 @@
         },
         "CHANNEL_GREETING_TOGGLE": {
           "LABEL": "Enable channel greeting",
-          "HELP_TEXT": "Automatically send a greeting message when a new conversation is created.",
+          "HELP_TEXT": "Automatically send a greeting message after the contact's first message in a conversation.",
```

Comment:
@Hricha-Shandily Please check this copy.

Reply:
How about: "**Instantly engage customers with automated greeting messages**."

P.S. Couldn't directly add it as a suggestion. We may have to open a new PR. cc: @iamsivin

Reply:
@Hricha-Shandily Will create a PR